virufy4/virufy4.github.io
# feat: add ufukui logo that links to either the japanese or english site

**Note from Amil:**
```
Can you please add the attached logo into website 4.0 (and later website 6.0)?
https://virufy.org/ja/oursupporters/

It should link here:
- Japanese: https://www.u-fukui.ac.jp/
- English: https://www.u-fukui.ac.jp/eng/

This is big news as getting logo use approval from a public university in Japan is very difficult, and brings much credibility to our cause.
```

**Steps to test functionality:**

Given the following supporter links in different locales, when clicking on the University of Fukui logo, it should open a new tab to the following links:

1. English: http://localhost:8000/en/oursupporters/ -> click on logo -> https://www.u-fukui.ac.jp/eng
2. Spanish: http://localhost:8000/es/oursupporters/ -> click on logo -> https://www.u-fukui.ac.jp/eng
3. Portuguese: http://localhost:8000/pt/oursupporters/ -> click on logo -> https://www.u-fukui.ac.jp/eng
4. Japanese: http://localhost:8000/ja/oursupporters/ -> click on logo -> https://www.u-fukui.ac.jp
